ray_on_golem/server/services/golem/golem.py

```diff
@@ -55,6 +58,9 @@ def __init__(self, websocat_path: Path, registry_stats: bool):
         self._stacks_locks: DefaultDict[(str, bool), asyncio.Lock] = defaultdict(asyncio.Lock)
         self._payment_manager: Optional[PaymentManager] = None
 
+        self._ssh_sentry_tasks: Dict[str, asyncio.Task] = {}
+        self._ssh_sentry_timeout: timedelta = DEFAULT_SSH_SENTRY_TIMEOUT
```

Comment:
The same comment for the different place - if this timeout is not in init, the code should use just the global constant from the file.

Reply:
fixed